sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -16,7 +16,7 @@ export const StudentRoutesConst = {
   NOTIFICATIONS: Symbol(),
   NOTIFICATIONS_SETTINGS: Symbol(),
   STUDENT_APPLICATION_SUMMARY: Symbol(),
-  STUDENT_EDIT_APPLICATION: Symbol(),
+  STUDEN_APPLICATION_DETAILS: Symbol(),
```

Comment:
Is it supposed to be "STUDENT_APPLICATION_DETAILS"?

Reply:
yes @andrewsignori-aot , it is the application details page, Application edit is a different page. That's what i understood from Figma. correct me if I am wrong @abschwenker

Reply:
I think you missed the T on Student :)

Reply:
ya, sorry . pls ignore the comment :)